flutter-for-android.md

```diff
@@ -1438,15 +1548,28 @@ class _SampleAppPageState extends State<SampleAppPage> {
 
 **How do I theme my app?**
 
-Flutter out of the box comes with a precise implementation of Material Design, which takes care of a lot of styling and theming needs that you would typically do.
-
-To take full advantage of MaterialDesign in your app, you can declare a top level widget `MaterialApp` as the entry point to your application. This is where you would declare routing and navigation as well
+Flutter out of the box comes with a precise implementation of Material Design, which takes care of a lot of styling and theming needs that you would typically do. Unlike Android where you declare themes in XML and then assign it to your application via AndroidManifest.xml, in Flutter you declare themes via the top level widget.
 
-[code]
-
-To customize the colors and styles of Material Design you can pass in a ThemeData object to the MaterialApp widget
+To take full advantage of MaterialDesign in your app, you can declare a top level widget `MaterialApp` as the entry point to your application. 
 
+To customize the colors and styles of Material Design you can pass in a ThemeData object to the MaterialApp widget, for example in the code below you can see the primary swatch is set to blue and all text selection color should be red.
```

Comment:
can you please line-break these at 80 chars?

Reply:
okay no problem, just grabbing lunch, will do it in an hour

Reply:
let me know if that's good now!